manifests/10-insights-runtime-extractor.yaml

```diff
@@ -93,6 +93,12 @@ spec:
           env:
             - name: CONTAINER_RUNTIME_ENDPOINT
               value: unix:///crio.sock
+          livenessProbe:
+            exec:
+              command:
+                - crictl
+                - info
+              periodSeconds: 10
```

Comment:
@tremes It will take 30 seconds for the container to be restarted (`failureThreshold` is 3 by default).

Should I reduce the `failureThreshold` to 1 (minimum or 2). I don't want to decrease the `periodSeconds` as it seems overkill to check that corner case too often. wdyt?